migration/v0.31/prepare/10-thanos-distrib-memory-limit.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+if ! yq_null sc .thanos.receiveDistributor.resources.limits.memory; then
+  log_info "- check if thanos distributor memory limit is less than 700Mi"
+  size=$(yq4 '.thanos.receiveDistributor.resources.limits.memory | sub("Mi","")' "${CK8S_CONFIG_PATH}/sc-config.yaml")
```

Comment:
Only issue here is if some has increased the memory limit to "1Gi", I assume that will error out in the `if ((size < 700))`

Reply:
yes it will fail and continue 

> [ck8s] v0.31/prepare/10-thanos-distrib-memory-limit.sh: - check if thanos distributor memory limit is less than 700Mi
> .../compliantkubernetes-apps/migration/v0.31/prepare/10-thanos-distrib-memory-limit.sh: line 12: ((: 1Gi: value too great for base (error token is "1Gi")
> [ck8s] v0.31/prepare/10-thanos-distrib-memory-limit.sh: - thanos distributor memory limit is 1Gi, will not update it

I will see how I can fix it

Reply:
please check again